charts/juicefs/templates/worker/configmap.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "juicefs.fullname" . }}-pod-state
```

Comment:
We recently identified a performance issue in our Kubernetes cluster where an excessive number of ConfigMaps caused instability in controller operations. 
To mitigate such resource contention challenges, I recommend implementing CRDs for more efficient configuration management.